core/src/test/java/org/elasticsearch/action/ingest/SimulatePipelineRequestTests.java

```diff
@@ -89,4 +86,10 @@ public void testSerializationWithXContentBwc() throws IOException {
             }
         }
     }
+
+    public void testUnsupportedReadFrom() {
+        SimulatePipelineRequest request = new SimulatePipelineRequest();
+        Exception exception = expectThrows(UnsupportedOperationException.class,() -> request.readFrom(null));
+        assertThat(exception.getMessage(), equalTo("usage of Streamable is to be replaced by Writeable"));
```

Comment:
OK cool. I was on the fence about it. figured I'd discuss it after the work has been done :) will happily remove